test/opf/opf.jl

```diff
@@ -45,14 +45,15 @@ include("acp.jl")
 include("dcp.jl")
 include("socwr.jl")
 include("sdpwrm.jl")
+include("sparse_sdpwrm.jl")
 include("ed.jl")
 
 # other tests
 include("quad_obj.jl")
 
 @testset "OPF" begin
     @testset "$(OPF)" for OPF in PGLearn.SUPPORTED_OPF_MODELS
-        if OPF == PGLearn.SDPOPF
+        if OPF in [PGLearn.SDPOPF, PGLearn.SparseSDPOPF]
```

Comment:
I'm OK with running CI only on networks with up to 30 buses, for all the formulations. We're kind of over-doing it going all the way up to 118